### ISSUE TITLE
bug: Uncaught (in promise) TypeError when including ion-grid in project with Ionic Core 4 delivered by CDN

### ISSUE THREAD

**Issue by pmikla14** [ionic-team/ionic#19930]
**Ionic version:**
[x] **4.x** (current version delivered by jsdelivr as of Nov 17, 2019)

**I'm submitting a ...** 
[x ] bug report
[ ] feature request

**Current behavior:**
When including the current Ionic version by CDN (jsdelivr) the `ion-grid` component does not work. As soon as I include `ion-grid` in the html file, the page does not show anything and the console shows the following errors:
```
Failed to load resource: the server responded with a status of 404 ()
TypeError: Failed to fetch dynamically imported module: https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/p-d9xvdh4f.entry.js
k @ p-86c36cf1.js:1
Uncaught (in promise) TypeError: Cannot read property 'isProxied' of undefined at p-86c36cf1.js:1

```
**Expected behavior:**
The grid should be shown. As soon as I get rid of the whole `ion-grid` component, the error disappears and the header is shown. 

**Steps to reproduce:**
As you can see below, only core ionic components are used. In my opinion this code should work and produce a page including a header and a 4-col-grid.

**Related code:**
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Ionic Core Test Application</title>
    <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.esm.js"></script>
    <script nomodule src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.js"></script>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core/css/ionic.bundle.css"/>
</head>
<body>
    <ion-app>
        <ion-content>
            <ion-header>
                <ion-toolbar color="primary">
                    <ion-title>Ionic 4 Test Application</ion-title>
                </ion-toolbar>
            </ion-header>
            <ion-grid>
                <ion-row>
                    <ion-col>
                        ion-col
                    </ion-col>
                    <ion-col>
                        ion-col
                    </ion-col>
                    <ion-col>
                        ion-col
                    </ion-col>
                    <ion-col>
                        ion-col
                    </ion-col>
                </ion-row>
            </ion-grid>
        </ion-content>
    </ion-app>
</body>
</html>
```

**Ionic info:** 
Ionic CLI is not used in this Project

**Comment by liamdebeasi** [comment]
Thanks for the issue. Which OS/browser are you testing this on? I am testing this in Chrome on macOS and am not getting this error.

CodePen example: https://codepen.io/liamdebeasi/pen/xxxmzaw

**Comment by pmikla14** [comment]
Hi, 

I’ve also been testing this on MacOS (Catalina 10.15.1) and Chrome (Version 78.0.3904.97). I updated now to the newest available Chrome (Version 78.0.3904.108) and now it’s working like a charm. Seemed to be a Chrome Bug or something…

Thanks for your support, this issue can be closed.
